traveltime-spec.yml

```diff
@@ -643,11 +664,14 @@ components:
       required:
         - scale_type
         - level
+        - square_size
```

Comment:
Is it correct to set this as `required` when it's only needed when `scale_type` is set to `coarse_grid`?

Reply:
Fix the mistake.

Reply:
I don't think this is right. It doesn't define the requirements of the different `scale_type`s. Here's a proposed solution, courtesy of ChatGPT:

```
    RequestLevelOfDetail:
      type: object
      required:
        - scale_type
      properties:
        scale_type:
          type: string
          enum:
            - simple
            - simple_numeric
            - coarse_grid
      discriminator:
        propertyName: scale_type
        mapping:
          simple: '#/components/schemas/SimpleScale'
          simple_numeric: '#/components/schemas/SimpleNumericScale'
          coarse_grid: '#/components/schemas/CoarseGridScale'
    SimpleScale:
      type: object
      properties:
        scale_type:
          type: string
          enum:
            - simple
        level:
          type: string
          enum:
            - lowest
            - low
            - medium
            - high
            - highest
      required:
        - level
    SimpleNumericScale:
      type: object
      properties:
        scale_type:
          type: string
          enum:
            - simple_numeric
        level:
          type: integer
          minimum: -20
          maximum: 2
      required:
        - level
    CoarseGridScale:
      type: object
      properties:
        scale_type:
          type: string
          enum:
            - coarse_grid
        square_size:
          type: integer
          minimum: 600
      required:
        - square_size
```

Please verify if it's correct.